test/functional/rpc_rawtransaction.py

```diff
@@ -438,21 +438,23 @@ def run_test(self):
 
         self.sync_all()
         inputs = [{ "txid" : txId, "vout" : vout['n'] }]
-        outputs = { self.nodes[0].getnewaddress() : Decimal("0.99999000") } # 1000 sat fee
+        outputs = { self.nodes[0].getnewaddress() : Decimal("0.98000000") } # 2000000 sat fee
```

Comment:
Why is the test changed at all?

Reply:
Let me explain it a bit in detail.

I wanted to test if a transaction with sufficiently high fee rate gets rejected if no `maxfeerate` was specified.
I could've created a new transaction with a high fee and run test with it, but I thought it would be nice if I can reuse the existing code so that the change becomes minimal.
The existing `rawTx`'s fee was too low for that, but there should be no difference if I change the related values altogether.
So I increased the fee from 1000 sat to 0.02 BTC, and other `maxfeerate` values accordingly.

Reply:
> I wanted to test if a transaction with sufficiently high fee rate gets rejected if no maxfeerate was specified.

This is tested in other functional tests already. (I presume you can find them by removing the failure in the source code, re-compile and re-run the tests)

Reply:
I changed my code as following, but couldn't find any failing test. Could you point the exact test you're referring to?
```diff
+ CFeeRate max_raw_tx_fee_rate{DEFAULT_MAX_RAW_TX_FEE_RATE * 1000000};
- CFeeRate max_raw_tx_fee_rate{DEFAULT_MAX_RAW_TX_FEE_RATE};
```

Reply:
You might be right. Going to take a look... :eyes:

Reply:
You are right. Sorry, you can keep your changes as they are.